src/Model.php

```diff
@@ -1565,15 +1628,15 @@ public function tryLoadAny()
      *
      * @return $this
      */
-    public function loadBy($field, $value)
+    public function loadBy(string $field_name, $value)
```

Comment:
should also fix $field_name type in method @param comment

Reply:
addressed.